core/src/main/java/com/google/errorprone/bugpatterns/UnusedMethod.java

```diff
@@ -303,10 +303,14 @@ private void handleMethodSource(MethodTree tree) {
         sym.getRawAttributes().stream()
             .filter(a -> a.type.tsym.getQualifiedName().equals(name))
             .findAny()
-            // get the annotation value array as a set of Names
-            .flatMap(a -> getAnnotationValue(a, "value"))
-            .map(
-                y -> asStrings(y).map(state::getName).map(Name::toString).collect(toImmutableSet()))
+            // get the annotation value array as a set of Names,
+            // normalizing unset value to the empty value
+            .map(a -> getAnnotationValue(a, "value")
+                .map(y -> asStrings(y).map(state::getName).map(Name::toString).collect(toImmutableSet()))
+                .orElse(ImmutableSet.of())
```

Comment:
nit: could this be `.orElse(ImmutableSet.of(sym.name.toString()))` instead of separately checking of names is empty below?